cmake/ThirdPartyPackagesKw.cmake

```diff
@@ -15,6 +15,13 @@ set(KNOWHERE_THIRDPARTY_DEPENDENCIES
         FAISS
 )
 
+if ( ${CMAKE_HOST_SYSTEM_NAME} MATCHES "Linux" AND KNOWHERE_WITH_DISKANN)
+    set(KNOWHERE_THIRDPARTY_DEPENDENCIES 
```

Comment:
in Knowhere's stype, we put new lib behind:

set(KNOWHERE_THIRDPARTY_DEPENDENCIES 
        ${KNOWHERE_THIRDPARTY_DEPENDENCIES}
        DiskANN
        )